engine/crash/src/backtrace_unwind.cpp

```diff
@@ -0,0 +1,172 @@
+// Copyright 2020-2022 The Defold Foundation
```

Comment:
Now it has the same name as the macos version, "libunwind" which is very confusing.